tests/flow-repo/declare_export/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1327,8 +1327,8 @@ var numberValue6 = require("ES6_ExportFrom_Intermediary2").numberValue1;
 var ap1: number = numberValue6;
 var ap2: string = numberValue6; // Error: number ~> string
 
-var numberValue2_renamed2 = require("ES6_ExportFrom_Intermediary2")
-  .numberValue2_renamed2;
+var numberValue2_renamed2 =
+  require("ES6_ExportFrom_Intermediary2").numberValue2_renamed2;
```

Comment:
I am afraid we will get a lot of issues on this

Reply:
Do you think many people prefer to have that property on a separate line? We'll see.

I'm done with this PR, BTW. What do you think about the rest of the changes? If you approve it (or somebody else), I'll merge it today.